src/wp-includes/functions.php

```diff
@@ -8735,6 +8735,16 @@ function is_wp_version_compatible( $required ) {
 	// Strip off any -alpha, -RC, -beta, -src suffixes.
 	list( $version ) = explode( '-', $wp_version );
 
+	if ( is_string( $required ) && substr_count( $required, '.' ) > 1 && str_ends_with( $required, '.0' ) ) {
+		$passed   = $required;
+		$required = preg_replace( '/\.0/', '', $passed, 1 );
+		wp_trigger_error(
+			__FUNCTION__,
+			/* translators: s: version string sent to function */
+			sprintf( __( '`%1$s` Not a valid WordPress version string. `%2$s` is assumed.' ), $passed, $required )
+		);
```

Comment:
Per notes on ticket -- will need changes to the tests.

Reply:
Are we planing on notifying the user in this manner or silently fixing the incorrect version?

Reply:
Silently fix due to the use of `@since x.x.0` throughout core. 

```suggestion
```